analyzers/tests/SonarAnalyzer.UnitTest/Helpers/TypeHelperTest.cs

```diff
@@ -89,5 +89,412 @@ public void Type_GetSymbolType_Alias()
             symbol.ToString().Should().Be("PropertyBag");
             type.ToString().Should().Be("System.Collections.Generic.Dictionary<string, object>");
         }
+
+        [DataTestMethod]
+        [DataRow("int")]
+        [DataRow("System.Int32")]
+        [DataRow("int?")]
+        [DataRow("System.Nullable<int>")]
+        [DataRow("CustomStruct")]
+        [DataRow("CustomRefStruct")]
+        [DataRow("RecordStruct")]
+        public void IsStruct_Simple(string type)
```

Comment:
To be aligned with the next name, that has `_False`
```suggestion
        public void IsStruct_Simple_True(string type)
```